eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/CDDLUtils.hs

```diff
@@ -30,18 +30,20 @@ import Cardano.Ledger.Serialization
     ToCBORGroup (..),
     groupRecord,
   )
-import Cardano.Prelude
-  ( Exception,
-    ExitCode (..),
-    forM_,
-    throwIO,
-  )
 import Codec.CBOR.Read (deserialiseFromBytes)
 import Codec.CBOR.Term (decodeTerm)
+import qualified Control.Exception as PException
+import Control.Monad.Trans as Trans
+  ( MonadIO,
+    liftIO,
+  )
 import qualified Data.ByteString.Base16.Lazy as Base16
 import qualified Data.ByteString.Lazy as BSL
 import Data.ByteString.Lazy.Char8 as Char8 (lines, unpack)
+import Data.Foldable (forM_)
 import Data.Typeable
+import GHC.Exception
+import GHC.IO.Exception (ExitCode (..))
```

Comment:
It's usually better to avoid `GHC.*` imports, unless specific internals unexported otherwise are needed

```suggestion
import Control..Exception
import System.Exit (ExitCode (..))
```